configure.ac

```diff
@@ -49,6 +49,14 @@ AC_ARG_VAR([OMR_CROSS_CONFIGURE],
 	Compiler and feature detection tests will be invalid. All options must be configured manually.
 	One of: yes,no. (Default: no)])
 
+AC_ARG_VAR([OMR_PRODUCT_DESCRIPTION], [The description of product integerating OMR. (Default: OMR 1.0)])
+AC_ARG_VAR([OMR_PRODUCT_NAME], [The name of product integerating OMR. (Default: OMR)])
+AC_ARG_VAR([OMR_PRODUCT_VERSION], [The version of product integerating OMR. (Default: 1.0)])
+
+OMR_PRODUCT_DESCRIPTION=${OMR_PRODUCT_DESCRIPTION:-"AC_PACKAGE_STRING AC_PACKAGE_URL"}
```

Comment:
They are defined by the macro [`AC_INIT`](https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Initializing-configure.html) used [here](https://github.com/eclipse/omr/blob/master/configure.ac#L23).

| MACRO | VALUE |
|------------------------|---------------|
| `AC_PACKAGE_STRING`   | `OMR`       |
| `AC_PACKAGE_VERSION` | `1.0`          |
| `AC_PACKAGE_STRING`   | `OMR 1.0` |
| `AC_PACKAGE_URL` | `<empty>` |